docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,15 @@
 ### What was the problem?
 
-### How did I fix it?
+### How did I solve?
 
-### How to test it?
+### How to manually test it?
 
 ### Review checklist
 
-* The PR resolves #INSERT_ISSUE_NUMBER
-* All new code is covered with unit tests
-* All new code was formatted with Prettier
-* Linting passes
-* Tests pass
-* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
-* Documentation has been added/updated
+- The PR resolves #INSERT_ISSUE_NUMBER
+- All new code is covered with unit tests
```

Comment:
Instead we can say code coverage is above 90% maybe ?

Reply:
Also we can make sure of that by adding as a requirement to merge the PR.

Reply:
In total coverage, we don't have 90%